### ISSUE TITLE
Accessibility issue with site-branding > custom-logo-link - missing discernible text (title attribute)

### ISSUE THREAD

**Issue by lydenyardley** [brainstormforce/astra#4177]
### Describe the bug:
After running the firefox extension webhint, I have found a few issues with your theme. Here is one that prevents screenreaders knowing what the content of the logo link is. As it's the main site logo, with no written content in the link, a title attribute is required

**Warning!** Links must have discernible text: Element has no title attribute
[https://yardley.digital/:17:126](view-source:https://yardley.digital/)

`<a href="https://yardley.digital/" class="custom-logo-link" rel="home" aria-current="page">`

### Expected behavior:
The site name logo should have a title attribute added to it that pulls the site name into it. So in my case:

`<a href="https://yardley.digital/" class="custom-logo-link" rel="home" aria-current="page">`
should be:
`<a href="https://yardley.digital/" class="custom-logo-link" rel="home" aria-current="page" title="Yardley Digital">`

### Screenshots:
![Screenshot from 2022-03-11 16-11-48](https://user-images.githubusercontent.com/2838799/157905082-791f5591-4281-4922-aa08-6993c1675f7b.png)

### WordPress version:
5.9.2

### Astra Theme version:
Latest

**Comment by lydenyardley** [comment]
Nudge

**Comment by lydenyardley** [comment]
Are you able to look at this team? Seems like a simple tweak...

**Comment by thebengalboy** [comment]
This issue is resolved and will be released soon- https://github.com/brainstormforce/astra/pull/4881

@lydenyardley We hope to release it with Astra version 4.0.2 or 4.0.3.

**Comment by lydenyardley** [comment]
@thebengalboy This still isn't resolved in 4.0.2. Is it definitely in the next build?

**Comment by thebengalboy** [comment]
@lydenyardley In addition to this, we have addressed the majority of accessibility issues and introduced a new accessibility field design control panel. This new feature will be available with the release of Astra version 4.1.0, which is currently in the quality assurance phase. You can anticipate its release to occur by the end of February or early March.

**Comment by lydenyardley** [comment]
Good man @thebengalboy Thank you very much!
